client/src/pages/Store/Store.jsx

```diff
@@ -10,118 +10,102 @@ import { MEDIA_URL } from '../../utils/config';
+  const additionalText = t('store.additionalText');
+  const mainTextMobile = t('store.mainTextMobile');
+  const additionalTextMobile = t('store.additionalTextMobile');
 
-    return (
-      <div className="section store-container">
-        <div className="store-wrapper">
-          <div className="store-main-text">{mainText}</div>
-          <div className="store-additional-text">{additionalText}</div>
+  return (
+    <div className="section store-container">
+      <div className="store-wrapper">
+        <div className="store-main-text">{mainText}</div>
+        <div className="store-additional-text">{additionalText}</div>
 
-          <div className="store-main-text-mobile">{mainTextMobile}</div>
-          <div className="store-additional-text-mobile">
-            {additionalTextMobile}
-          </div>
-          <div className="carousel-container">
-            <Slider {...settings}>
-              {merches.map(merch => (
-                <div key={merch.id} className="carousel-block">
-                  <div className="carousel-info__merch">
-                    <div className="wrapper">
-                      <div className="ellipse" />
-                      <ImageLoader
-                        className="merch-logo"
-                        src={`${MEDIA_URL}/${merch.avatar_url}`}
-                        alt="merch"
-                      />
-                    </div>
+        <div className="store-main-text-mobile">{mainTextMobile}</div>
+        <div className="store-additional-text-mobile">
+          {additionalTextMobile}
+        </div>
+        <div className="carousel-container">
+          <Slider {...settings}>
+            {merches.map(merch => (
+              <div key={merch.id} className="carousel-block">
+                <div className="carousel-info__merch">
+                  <div className="wrapper">
+                    <div className="ellipse" />
+                    <ImageLoader
+                      className="merch-logo"
+                      src={`${MEDIA_URL}/${merch.avatar_url}`}
+                      alt="merch"
+                    />
                   </div>
                 </div>
-              ))}
-            </Slider>
-          </div>
-          <div className="store-buttons-mobile">
-            <MerchSize
-              changeMerchAttr={changeMerchAttr}
-              size={order.size}
-              merches={merches}
-            />
-            <MerchBuy handleDialog={handleDialog} cost={order.cost} />
-          </div>
+              </div>
+            ))}
+          </Slider>
+        </div>
+        <div className="store-buttons-mobile">
+          <MerchSize
+            changeMerchAttr={changeMerchAttr}
+            size={order.size}
+            merches={merches}
+          />
+          <MerchBuy handleDialog={handleDialog} cost={order.cost} />
         </div>
       </div>
-    );
-  }
+    </div>
+  );
 }
 
 export default withTranslation('')(Store);
```

Comment:
withTranslation you can overwrite to useTranslation hook like in another components
see here https://react.i18next.com/latest/usetranslation-hook